### ISSUE TITLE
Semicolon color is too "dark" in dark mode (very hard to see)

### ISSUE THREAD

**Issue by keitzer** [MateoCerquetella/xcode-theme#7]
**Is your feature request related to a problem? Please describe.**
I want a way to change the semicolon color. I'm currently using VS Code with C# (specifically with Unity), and using the Xcode theme so far has been great as that's what i'm used to with Swift/Xcode when working with iOS. -- however with the semicolon, the contrast is simply too low in the dark mode to effectively see the punctuation to know if i've typed it or not.

After a few attempts of trying to run an override of the punctuation I can't seem to get the semicolon to change on me. I'm using the `Xcode Default Dark Customized Version` and yet it still isn't working.

**Describe the solution you'd like**
A way to set the semicolon color (and possibly other properties?), ideally to a higher contrast (eg: white)

**Describe alternatives you've considered**
This is what I've tried to no avail (as well as a few variants on this, based on google searches):
(in `settings.json`)
```

    "editor.tokenColorCustomizations": {
        "textMateRules": [
          {
            "scope": [
              "punctuation.terminator",
              "punctuation",
            ],
            "settings": {
              "foreground": "#ffffff",
            }
          }
        ]
      }
```

## Consider this
I'm typing with the following scenario.. notice in the line with the cursor, it is hard to see if the semicolon has been typed (the line has been duplicated for demonstration purposes, but hopefully this makes the visual clear):

### CURRENT
![image](https://user-images.githubusercontent.com/8125837/175180320-08f36503-8951-496f-b312-0af509b3e1de.png)

### WHAT IT LOOKS LIKE (with a slight blur to emphasize contrast + how it looks to me with my eyes):
![image](https://user-images.githubusercontent.com/8125837/175180905-374f5964-270f-4347-a369-3e452b469eb2.png)

### PROPOSED:
![image](https://user-images.githubusercontent.com/8125837/175180936-0b5875f9-9f58-44ec-9d5a-88f8f95a5d89.png)

### SIMILAR BLUR APPLIED TO PHOTO:
![image](https://user-images.githubusercontent.com/8125837/175180958-82274d80-ff59-4141-9143-e7a61b9131e2.png)

**Comment by MateoCerquetella** [comment]
@keitzer Thanks for the report on this issue, As you know Swift the  Apple programming language don't use semicolons on their language, so in my case I used this "trick" to make it more invisible, but after some new perspective I'm gonna change the semicolon icon to the new hex color #FFFFFF

**Comment by MateoCerquetella** [comment]
Published version 4.0.0 with new major changes, gonna close this issue, if there is any doubt you can reopen it 👍🏻
